### PR TITLE
Update to Wagtail 5.2

### PR DESCRIPTION
**Pull request by zerolab**
And Django 4.2 + related package updates.
Replaces modeladmin usage with `ModelViewSet` for packages and taxonomy
Does a bit of tidy-up. The commits are fairly atomic

### CODE REVIEW

File: wagtailio/packages/viewsets.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class PackageAdmin(BaseModelViewSet):
+    model = Package
+    form_fields = ["publish", "title", "slug", "repo_description"]
+
+
+class PackagesAdminGroup(ModelViewSetGroup):
+    menu_label = "Package"
+    menu_icon = "folder-open-inverse"
```
Comment (William-Blackie) [inline wagtailio/packages/viewsets.py:37]: Nitpick: We're using the same iconography as the page(s) button, as these are landmarks on a page it feels a little strange to me? 

Perhaps we could use `form` or `tasks`? [options we could use](https://docs.wagtail.org/en/latest/advanced_topics/icons.html)

**Review by William-Blackie — APPROVED** [review]
Hey Dan, great work here. I'm struggling to provide you constructive feedback specific the PR, it looks good and works from testing. 

There is however, two side quests, if you choose to accept them.

1. Alter the Dockerfile to export the bash aliases again ( I miss `djrun and dj`). The file is there, it was just never added.
2. Alter the Dockerfile to run `createcachetable` upon build, it's a minor issues that can be resolved by a developer, but would make the initial setup painless. 

These are however, outside the initial scope of this PR, so I will not block this PR. 

Thanks - Will.

File: fabfile.py [hunk 2]
```diff
@@ -34,9 +34,7 @@
 
 
 def dexec(cmd, service="web"):
-    return local(
-        "docker-compose exec -T {} bash -c {}".format(quote(service), quote(cmd))
-    )
+    return local(f"docker-compose exec -T {quote(service)} bash -c {quote(cmd)}")
```
[the remark this answers is not in the document]
Comment (zerolab) [reply fabfile.py:37]: I wanted to keep that in a separate PR

File: wagtailio/packages/viewsets.py (hunk 1 above)
Comment (zerolab) [reply wagtailio/packages/viewsets.py]: changed to https://fontawesome.com/icons/cube?f=classic&s=solid

File: fabfile.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (zerolab) [reply fabfile.py:37]: opened #449

**Pull request merged**